subsys/bluetooth/mesh/Kconfig

```diff
@@ -447,6 +451,26 @@ config BT_MESH_MSG_CACHE_SIZE
 	  Setting this value to a very large number can impact the processing time
 	  for each received network PDU and increases RAM footprint proportionately.
 
+config BT_MESH_RELAY_BUF_COUNT
```

Comment:
Maybe instead move `endif # BT_MESH_RELAY` above `config BT_MESH_RELAY_BUF_COUNT`? Then only changed text will be visible.